### ISSUE TITLE
psv  tags ignored

### ISSUE THREAD

**Issue by grischard** [GIScience/openrouteservice#562]
#### Here's what I did

https://maps.openrouteservice.org/directions?n1=49.6092&n2=6.124652&n3=17&a=49.609577,6.128503,49.610264,6.123964&b=4c&c=0&k1=en-US&k2=km
---
#### Here's what I got

Routing over Marie-Thérèse, Joseph II and back over Monterey

---
#### Here's what I was expecting

Turn right onto Boulevard Royal (`oneway:psv = no`) then turn left onto Avenue Monterey (`access=no` with `psv=yes`)

---
#### Here's what I think could be improved

Use the `psv` tags for bus routing.

**Comment by nilsnolde** [comment]
Yes, you're right. In the end those HGV sub-profiles are unfortunately fairly poorly done. 

We're migrating our code and at this point it would be futile to try and patch those profiles. We'll be looking at supporting a subset after the migration, but that's more towards end of the year.
